### PR TITLE
Branding guidelines

### PR DESCRIPTION
**Pull request by maxwofford**

### CODE REVIEW

File: app/views/static_pages/branding.html.erb [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<% title 'HCB Brand Guidelines' %>
+
+<div class="container container--md">
```
Comment (lachlanjc) [inline app/views/static_pages/branding.html.erb]: Prefer removing this & adding `container_md` at the top

File: app/views/static_pages/branding.html.erb [hunk 2]
```diff
@@ -0,0 +1,54 @@
+<% title 'HCB Brand Guidelines' %>
```
Comment (lachlanjc) [inline app/views/static_pages/branding.html.erb]: Prefer spelling out “Hack Club Bank” on all user-facing pages

File: app/views/static_pages/branding.html.erb [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    <span class="muted h3 regular block">
+      How and where to refer to Hack Club Bank.
+    </span>
+  </h1>
+
+  <h2>
+    Badges
+  </h2>
+
+  <p>
```
Comment (lachlanjc) [inline app/views/static_pages/branding.html.erb]: Can be removed, no?

File: app/views/static_pages/branding.html.erb [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+          <%= link_to 'Download SVG', "/brand/hcb-logo-#{logo[:name].parameterize}.svg", class: 'btn bg-info mb1 w-auto', target: :_blank %>
+          <%= link_to 'Download PNG', "/brand/hcb-logo-#{logo[:name].parameterize}.png", class: 'btn bg-info', target: :_blank %>
+        </div>
+      </div>
+    <% end %>
+  </div>
+
+  <h2>
+    Legal relationship<br/>
+    <span class="muted h4 regular">
```
Comment (lachlanjc) [inline app/views/static_pages/branding.html.erb]: Prefer putting caption outside the heading &/or add `sans` class to use system font for all body-or-smaller-size UI text.

Also, rich quote :)

File: app/views/static_pages/branding.html.erb [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+      You'll want to mention this for tax purposes.
+    </span>
+  </h2>
+
+  <p>
+  MVHacks is fiscally sponsored by Hack Club Bank, a project by the The Hack
```
Comment (lachlanjc) [inline app/views/static_pages/branding.html.erb]: If you have an event, a really nice touch would be putting one of your event names here.

File: public/brand/hcb-logo-original-dark.svg [hunk 6]
```diff
@@ -0,0 +1,18 @@
+<svg width="256" height="64" viewBox="0 0 256 64" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (lachlanjc) [inline public/brand/hcb-logo-original-dark.svg]: Ideally would run `svgo` on these to compress.

**Review by lachlanjc — COMMENTED** [review]
This would be an awesome addition to Bank. Any timeline on launching this?

**Comment by lachlanjc** [comment]
Why is this closed? I see routine questions about the badges this page answers

**Comment by maxwofford** [comment]
Second that. Obv bank team's call here, but I see a need that this fulfills

**Comment by thesephist** [comment]
I've only ever gotten one question about the badges in my last month of doing Bank stuff, so I assumed this wasn't a big need.

If the comments in the PR are all we need, we can try to push it through. I don't have strong feels about this one, just wanted to clean up PRs that weren't getting love

**Comment by thesephist** [comment]
What's the reason people ask for branding? Is it usually to put it on their site?

I've only ever had one organizer ask about any kind of logo placement, so I have very little to go off of

File: app/views/static_pages/branding.html.erb (hunk 1 above)
Comment (thesephist) [reply app/views/static_pages/branding.html.erb]: Can you elaborate?

**Review by thesephist — APPROVED** [review]
just gonna approve it b/c its fairly low risk, will address comments after merge :)

**Pull request merged**